### PR TITLE
Correct unit-test.md call to reconcile client

### PR DESCRIPTION
**Pull request by qinqon**
The field of the client in the reconcile is client not cl

**Description of the change:**

**Motivation for the change:**

### CODE REVIEW

File: doc/user/unit-testing.md [hunk 1]
```diff
@@ -254,4 +254,4 @@ func TestMemcachedController(t *testing.T) {
 [doc-reconcile]: https://godoc.org/sigs.k8s.io/controller-runtime/pkg/reconcile#Reconciler
 [code-test-example]: https://github.com/operator-framework/operator-sdk-samples/blob/master/memcached-operator/pkg/controller/memcached/memcached_controller_test.go#L25
 [user-guide]: ../user-guide.md#register-with-the-managers-scheme
-[ocp-doc-v1-route]: https://docs.openshift.com/container-platform/3.11/rest_api/apis-route.openshift.io/v1.Route.html
\ No newline at end of file
+[ocp-doc-v1-route]: https://docs.openshift.com/container-platform/3.11/rest_api/apis-route.openshift.io/v1.Route.html
```
Comment (LiliC) [inline doc/user/unit-testing.md]: What changed here, I can't see a difference 🤔

Comment (qinqon) [reply doc/user/unit-testing.md]: Humm let me checkout that, I just one to change the cl vs client

**Pull request merged**